WooCommerce/Classes/POS/Presentation/PointOfSaleAssets.swift

```diff
@@ -9,7 +9,7 @@ enum PointOfSaleAssets: CaseIterable {
     case exit
     case getSupport
     case removeCartItem
-    case dismissProductsBanner
+    case xClose
```

Comment:
Renamed this since it is being used in various places so I wanted to make it a bit more generic

Reply:
We spoke about with @joshheald offline but it would make sense to use SF symbols rather than images for most of these. `xmark` represents the same symbol, so we could use `Image(systemName: "xmark")`.

Reply:
See #13458